packages/desktop-client/src/components/reports/ReportSummary.tsx

```diff
@@ -167,7 +169,14 @@ export function ReportSummary({
           </PrivacyFilter>
         </Text>
         <Text style={{ fontWeight: 600 }}>
-          Per {(ReportOptions.intervalMap.get(interval) || '').toLowerCase()}
+          <Trans>
+            Per{' '}
+            {{
+              interval: (
+                ReportOptions.intervalMap.get(interval) || ''
```

Comment:
IIRC this map contains hard-coded English strings. We might need a bigger migration before translating this one

Reply:
good shout: 586012f47098568620fa7ff754cbc0fa574a815c